pkg/code-generator/schemagen/generator.go

```diff
@@ -22,6 +22,18 @@ type ValidationSchemaOptions struct {
 
 	// Tool used to generate JsonSchemas, defaults to protoc
 	JsonSchemaTool string
+
+	// Whether to remove descriptions from validation schemas
+	// Default: false
+	//
+	// NOTE: I'd prefer a positive field name (ie includeDescriptions)
+	//	but I wanted to avoid changing the default behavior
+	RemoveDescriptionsFromSchema bool
```

Comment:
[nit] What if `MaxDescriptionCharacters` was 0 and `RemoveDescriptionsFromSchema` is true? Will one override the other? Would be nice to have this in the comment to quickly grok the behavior.

Reply:
Yeah good call